src/stories/layouts/Wizard/Wizard.js

```diff
@@ -85,21 +90,26 @@ export const Wizard = ({ children, labels }) => {
  *   <Fragment WizInstance={WizInstance} />
  * )
  */
-const Controls = ({ experimentId, currentStep, WizInstance }) => (
+const Controls = ({ experimentId, currentStep, WizInstance, showNext }) => (
   <Fragment>
     <div className="wizard__controls">
       {currentStep !== 1 &&
         <button className={'wizard__button button--link'}
           onClick={WizInstance.previousStep}
         >Go Back</button>
       }
-      {currentStep !== WizInstance.totalSteps ?
+      {(currentStep !== WizInstance.totalSteps && showNext) ?
         <button className={'wizard__button button button--tertiary'}
           onClick={WizInstance.nextStep}
-        >Next</button> :
-        <Link className={'wizard__button button button--tertiary'}
-          to={`/study/${experimentId}/user/123`}
-        >Completet Study</Link>
+        >Next</button> : currentStep === WizInstance.totalSteps ?
+          <Link className={'wizard__button button button--tertiary'}
+            to={`/study/${experimentId}/user/123`}
+          >Complete Study</Link> :
+          <button className={'wizard__button ' +
+            'button ' +
+            'button--tertiary ' +
+            'button--disabled'}
+          >Next</button>
```

Comment:
I would change this to this:
```
{currentStep === WizInstance.totalSteps ?
        <Link className={'wizard__button button button--tertiary'}
          to={`/study/${experimentId}/user/123`}
        >Complete Study</Link> :
        <button className={'wizard__button button button--tertiary'}
          onClick={WizInstance.nextStep} disabled={!showNext}
        >Next</button>
      }
```

You can use the disabled attribute to prevent the button from working and not needing extra logic.